src/util/std_expr.h

```diff
@@ -4622,7 +4625,9 @@ class class_method_descriptor_exprt : public nullary_exprt
     return get(ID_C_class);
   }
 
-  const irep_idt &get_base_name() const
+  /// The name of the method to which this expression is applied as would be
+  /// seen in the source code. For example this could be - `toString`.
+  const irep_idt &base_method_name() const
```

Comment:
:pick: for me, `method_base_name` reads a little more naturally

Reply:
I think it’s a question of whether it’s the base of the method name or the name of the method base.

Reply:
"base/basis of the method's name". We aren't dealing with a method called "base" here. I wanted to insert the word "method" in order to make it clear that this particular name relates to the method, rather than somehow to the class or the expression as a whole. I don't really see that either ordering would be clearer than the other.